decoder/src/log.rs

```diff
@@ -27,12 +27,13 @@ pub fn log_defmt(
     line: Option<u32>,
     module_path: Option<&str>,
 ) {
-    let level = match frame.level() {
-        crate::Level::Trace => Level::Trace,
-        crate::Level::Debug => Level::Debug,
-        crate::Level::Info => Level::Info,
-        crate::Level::Warn => Level::Warn,
-        crate::Level::Error => Level::Error,
+    let level: Option<Level> = match frame.level() {
+        crate::Level::Trace => Some(Level::Trace),
+        crate::Level::Debug => Some(Level::Debug),
+        crate::Level::Info => Some(Level::Info),
+        crate::Level::Warn => Some(Level::Warn),
+        crate::Level::Error => Some(Level::Error),
+        crate::Level::Println => None,
```

Comment:
If we keep the fallback to `Info`, this branch could just be `=> Level::Info` and we'd remove the `Option`